examples/pylab_examples/arctest.py

```diff
@@ -1,18 +1,17 @@
-#!/usr/bin/env python
-from pylab import *
-
+import matplotlib.pyplot as plt
+import numpy as np
 
 def f(t):
     'a damped exponential'
-    s1 = cos(2*pi*t)
-    e1 = exp(-t)
-    return multiply(s1, e1)
+    s1 = np.cos(2*np.pi*t)
+    e1 = np.exp(-t)
+    return np.multiply(s1, e1)
```

Comment:
heh, don't know why `multiply` was ever used here. Let's just replace that with `s1 * e1`.